dallinger/default_configs/global_config_defaults.txt

```diff
@@ -33,7 +33,7 @@ chrome-path = /Applications/Google Chrome.app/Contents/MacOS/Google Chrome
 
 [Heroku]
 clock_on = False
-heroku_python_version = 3.10.2
+heroku_python_version = 3.10
```

Comment:
I don't think this will work as you need to specify a patch version. 
It should give you this error:
`Requested runtime 'python-3.10' is not available for this stack (heroku-22).`

Reply:
I think a solution would be to leave the `heroku_python_version` config parameter unspecified and change the code in a way to default automatically to the latest supported Python version on Heroku. See here: https://devcenter.heroku.com/articles/python-support#specifying-a-python-version

Reply:
I am wondering if this is resolved.

Reply:
Ah sorry forgot to reply to this comment! You're right -- I thought I had tested it, but apparently not. I took a different strategy now, to save us from having to update the code all the time. I now removed the default runtime.txt, so that if you leave heroku_python_version blank then it'll let Heroku choose the Python version. Does that make sense to you?